sdk/src/ordinal-api/index.ts

```diff
@@ -432,8 +472,30 @@ export class OrdinalsClient {
      * console.log("Output:", output);
      * ```
      */
-    async getInscriptionsFromOutPoint(outPoint: OutPoint): Promise<OutputJson> {
-        return await this.getJson<OutputJson>(`${this.basePath}/output/${OutPoint.toString(outPoint)}`);
+    getInscriptionsFromOutPoint(outPoint: OutPoint): Promise<OutputJson> {
+        return this.getJson<OutputJson>(`${this.basePath}/output/${OutPoint.toString(outPoint)}`);
+    }
+
+    /**
+     * Retrieves inscriptions based on the address.
+     * @param {String} address - The Bitcoin address to check.
+     * @param {('cardinal' | 'inscribed' | 'runic')} [type] - Optional type of UTXOs to be returned. If omitted returns all types.
+     * @returns {Promise<OutputJson>} A Promise that resolves to the inscription data.
+     *
+     * @example
+     * ```typescript
+     * const client = new OrdinalsClient("regtest");
+     * const address: string = "enter_address_here";
+     * const type: 'cardinal' | 'inscribed' | 'runic' = "enter_type_here";
+     * const output = await client.getUTXOFromAddress(address, type?);
+     * console.log("Output:", output);
+     * ```
+     */
+    getInscriptionsFromAddress(address: string, type?: 'cardinal' | 'inscribed' | 'runic'): Promise<OutputJson[]> {
```

Comment:
This naming is a bit odd, especially when called with `type = cardinal`, since it does exactly the opposite of what the name suggests: it then returns utxos that _don't_ have inscriptions.

Maybe something like `getOutputsByType` and possibly make `type` non-optional (add `any` option).

Reply:
updated